content/_index.en.html

```diff
@@ -20,7 +20,7 @@ <h2 class="mb-5">Open Source Railway Designer</h2>
 
 
 {{% blocks/lead color="primary" %}}
-An open source web application for railway infrastructure design, capacity analysis, timetabling, and more
+An open source web application for railway infrastructure design, capacity analysis, timetabling and more!
```

Comment:
```suggestion
An open source web application for railway infrastructure design, capacity analysis, timetabling, and more!
```
Can't we keep this comma here ?